core/templates/pages/header_js_libs.html

```diff
@@ -1,4 +1,7 @@
-<script async src="https://www.googletagmanager.com/gtag/js"></script>
+<% if (CAN_SEND_ANALYTICS_EVENTS) { %>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=<%= GA_ANALYTICS_ID %>"></script>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=<%= UA_ANALYTICS_ID %>"></script>
+<% } %>
```

Comment:
Can you please add an explanatory comment about the fact that different code is downloaded with each link (based on the id).

Reply:
Done!